.github/workflows/release.yaml

```diff
@@ -149,4 +173,54 @@ jobs:
           name: v${{ env.VERSION }}
           draft: true
           prerelease: ${{ startsWith(env.VERSION, '0') }}
-          files: artifacts/*
\ No newline at end of file
+          files: artifacts/*
+
+  publish-registry-image:
+    name: Build and publish Docker package
+    runs-on: ubuntu-latest
+    needs: [ prepare ]
+    if: ${{ needs.prepare.outputs.registry_published == 'not found' }}
+    permissions:
+      contents: read
+      packages: write
+    env:
+      VERSION: ${{ needs.prepare.outputs.version }}
```

Comment:
This seems to be redundant now.

Reply:
Yep, you're right. Thanks for spotting that one.